src/renderer/components/Login.js

```diff
@@ -19,14 +20,25 @@ class Login extends React.Component {
         showAdvanced: false,
         showPasswordMail: false,
         showPasswordSend: false
-      }
+      },
+      progress: 0
     }
+    this._updateProgress = this._updateProgress.bind(this)
+    ipcRenderer.on('DC_EVENT_CONFIGURE_PROGRESS', this._updateProgress)
 
     this.handleCredentialsChange = this.handleCredentialsChange.bind(this)
     this.handleSubmit = this.handleSubmit.bind(this)
     this.renderPasswordInput = this.renderPasswordInput.bind(this)
   }
 
+  componentWillUnmount () {
+    ipcRenderer.removeListener('DC_EVENT_CONFIGURE_PROGRESS', this._updateProgress)
+  }
+
+  _updateProgress (ev, progress) {
+    this.setState({ progress })
```

Comment:
Does this actually update it in the correct object path? The progress variable is in `state.ui.progress`, this seems like it puts it in `state.progress`. I assume it's wrong because for me the progress bar never updates/shows any green progress (only the grey loading one).

Reply:
I didn't set the color, thats why it is grey

Reply:
changed it to blue